diff --git a/src/detection/terminalshell/terminalshell_linux.c b/src/detection/terminalshell/terminalshell_linux.c
index 6fb5cc00..e9b37066 100644
--- a/src/detection/terminalshell/terminalshell_linux.c
+++ b/src/detection/terminalshell/terminalshell_linux.c
@@ -174,6 +174,11 @@ static void getTerminalShell(FFTerminalShellResult* result, pid_t pid)
 
     result->terminalPid = (uint32_t) pid;
     ffStrbufSetS(&result->terminalProcessName, name);
+    while(ffStrbufStartsWithC(&result->terminalProcessName, '.') && ffStrbufEndsWithS(&result->terminalProcessName, "-wrapped"))
+    {
+        ffStrbufSubstrAfter(&result->terminalProcessName, 0);
+        ffStrbufSubstrBefore(&result->terminalProcessName, result->terminalProcessName.length - sizeof("-wrapped") + 1);
+    }
     getProcessInformation(pid, &result->terminalProcessName, &result->terminalExe, &result->terminalExeName);
 }
 
